anomaly/anomaly_detector.py

```
@@ -107,45 +113,51 @@ async def _load_model_fallback() -> bool:
-                fallback=_load_model_fallback
+        try:
+            result = loop.run_until_complete(
+                _model_loader_cb.call(
+                    _load_model_with_retry,  # Retry wrapper
+                    fallback=_load_model_fallback,
+                )
             )
-        )
-        return result
-        
+            return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)